intellij-plugin-structure/tests/src/test/kotlin/com/jetbrains/plugin/structure/hub/mock/HubPluginMockTest.kt

```diff
@@ -1,43 +1,20 @@
 package com.jetbrains.plugin.structure.hub.mock
 
-import com.jetbrains.plugin.structure.base.utils.contentBuilder.buildZipFile
 import com.jetbrains.plugin.structure.base.plugin.PluginCreationFail
 import com.jetbrains.plugin.structure.base.plugin.PluginCreationSuccess
 import com.jetbrains.plugin.structure.base.plugin.PluginProblem
 import com.jetbrains.plugin.structure.hub.HubPlugin
 import com.jetbrains.plugin.structure.hub.HubPluginManager
+import com.jetbrains.plugin.structure.intellij.utils.URLUtil
 import org.junit.Assert
 import org.junit.Assert.assertEquals
-import org.junit.Rule
 import org.junit.Test
-import org.junit.rules.TemporaryFolder
 import java.io.File
 
 class HubPluginMockTest {
-
-  @Rule
-  @JvmField
-  val temporaryFolder = TemporaryFolder()
-
-  private val mockHubPluginManifestContent = perfectHubPluginBuilder.modify {
-    key = "cat-widget"
-    name = "Pets"
-    version = "1.1.1"
-    author = "Michael Jackson <mj@gmail.com>"
-    homeUrl = "https://github.com/mariyadavydova/youtrack-cats-widget"
-    description = "Funny cats and dogs for your Dashboard!"
-    iconUrl = "images/cat_purr.png"
-    dependencies = mapOf("Hub" to ">=2018.2")
-    products = mapOf("Hub" to "^2018.2", "YouTrack" to "^2018.2")
-  }
-
   @Test
   fun `hub plugin`() {
-    val pluginFile = buildZipFile(temporaryFolder.newFile("plugin.zip")) {
-      file(HubPluginManager.DESCRIPTOR_NAME) {
-        mockHubPluginManifestContent
-      }
-    }
+    val pluginFile = URLUtil.urlToFile(this::class.java.getResource("/hub/hub-mock.zip"))
```

Comment:
Please revert this. We don't want to pollute git repository with binaries. use `buildZipFile`

And this test doesn't check that `HubPlugin.icons` are correct. Use 
```
buildZip(...) {
   dir("images") {
     file("icon.png", SOME_CONTENT)
   }
}
```

and later check that `HubPlugin.icons.single()` has content of SOME_CONTENT